src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PermissionGrid/style.less

```diff
@@ -1,4 +1,12 @@
 :local(.permissionGrid) {
     margin-top: -50px;
     margin-bottom: 20px;
+    .dnn-suggestion {
+        .rw-popup-container {
+            .rw-list-option {
```

Comment:
Also needs `white-space: nowrap` to work on Safari

Reply:
thx for this comment.